src/training/data.py

```diff
@@ -361,5 +363,34 @@ def get_data(args, preprocess_fns, epoch=0):
     if args.val_data:
         data["val"] = get_wds_dataset(
             args, preprocess_val, is_train=False)
+    if args.image_data:
+        embeddings_images = EmbeddingReader(
+            embeddings_folder=f'{args.image_data}/img_emb/',
+            file_format='npy'
+        )
+        embeddings_txt = EmbeddingReader(
+            embeddings_folder=f'{args.image_data}/text_emb/',
+            file_format = 'npy'
+        )
+        worker_start_indices = torch.linspace(0, 1e9, args.world_size, dtype=torch.long)
```

Comment:
instead of 1e9 shouldn't this be something like args.train_num_samples or something like that?